go-fuzz-dep/main_libFuzzer.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2015 Dmitry Vyukov. All rights reserved.
```

Comment:
Note: the copyright message has changed on HEAD to a more appropriate one.

Reply:
Done